api/src/main/java/edu/wpi/first/shuffleboard/api/components/EditableLabel.java

```diff
@@ -12,12 +14,17 @@
   private final PseudoClassProperty editing = new PseudoClassProperty(this, "editing");
   private final StringProperty text = new SimpleStringProperty(this, "text");
 
+
+
```

Comment:
Surprised checkstyle did not find this.  There are too many empty lines here.  One will do.